clic.adoc

```diff
@@ -387,9 +387,45 @@ interrupt input.
 
 == smclic M-mode CLIC extension
 
-=== CLIC Memory-Mapped Registers
+Implementations can provide access to CLIC registers clicinttrig[i], clicintip[i], clicintie[i], clicintattr[i], clicintctl[i] 
+by memory-mapped registers or by using the Indirect CSR Access method (Smcsrind/Sscsrind).  
+The clic register accesses method available to software is indicated by the parameters CLICMEMMAPPED and CLICCSRINDIRECT.
+
+=== CLIC Register Access via Indirect CSR Access 
+
+The Indirect CSR Access allocated range for CLIC is TBD.  These address provided below are relative to the base of that allocated range.
```

Comment:
Spurious space after TBD.
I think it is clearer to refer to select values rather than addresses. Select values are a kind of address, but its a more precise term in this context.